recipes/flatbuffers/all/conanfile.py

```diff
@@ -47,15 +47,15 @@ def _cmake_new_enough(self, required_version):
             import re
             from io import StringIO
             output = StringIO()
-            self.run("cmake --version", output=output)
+            self.run("cmake --version", output)
```

Comment:
This can be removed with the updated cmake recipe with portable binaries :)

Reply:
Yes, please!

Reply:
The motivation behind this check is to avoid to ends up downloading tones of cmake versions, since there is quickly a divergence between various cmake versions hardcoded in these recipes. Is conan client plan to add a conf or something similar so that users can specify if they have cmake and which version?

Reply:
Very very few people actually have asked for anything near this and it adds too much complexity.

So we are going to be removing this. 

- Builds should be reproducible and these add  more variables then we care to see
- cci serves as an example and this is not a good practice